src/openvic-simulation/misc/Modifier.hpp

```diff
@@ -115,7 +115,7 @@ namespace OpenVic {
 		 * so instead we use an IdentifierInstanceRegistry (using std::unique_ptr's under the hood).
 		 */
 	private:
-		IdentifierInstanceRegistry<ModifierEffect> IDENTIFIER_REGISTRY(modifier_effect);
+		CaseInsensitiveIdentifierInstanceRegistry<ModifierEffect> IDENTIFIER_REGISTRY(modifier_effect);
 		string_set_t complex_modifiers;
```

Comment:
```suggestion
		case_insensitive_string_set_t complex_modifiers;
```
This should also be case insensitive

Reply:
Thanks, I totally missed this.